src/store/TranscriptionsStore.js

```diff
@@ -20,6 +20,9 @@ const Extension = types.refinement(types.map(Frame), snapshot => {
   return Ramda.all(Ramda.startsWith('frame'), Ramda.keys(snapshot))
 })
 
+let patchQueue = []
```

Comment:
Wait a goshdarn second... is `patchQueue` just floating around as a global variable, instead of an array in the TranscriptionStore's model?

Reply:
I'll see if it makes more sense to put this in the store model